sg_covid_impact/sic.py

```diff
@@ -0,0 +1,43 @@
+import requests
+import os
+import logging
+import sg_covid_impact
+import re
+import pandas as pd
+
+project_dir = sg_covid_impact.project_dir
+
+def url():
+    return sg_covid_impact.config['fetch_urls']['sic_taxonomy'] 
+
+def extract_sic_code_description(table, var_name, var_name_tidy):
+    """Extracts codes and descriptions from SIC table
+    Args:
+        table (pandas.DataFrame): summary of the SIC taxonomy
+        var_name (str): level of SIC we want to extract a lookup for
+        var_name_tidy (str): name for the variable code
+    Returns:
+        A lookup between the variable codes and their description
+    """
+    loc = list(table.columns).index(var_name)  # Find the location of class
+    select = table.iloc[:, [loc, loc + 1]].dropna()  # Extract variable & description
+    select.columns = [var_name, "description"]
+
+    select[var_name_tidy] = [re.sub(r"\.", "", str(x)) for x in select[var_name]]
```

Comment:
As the output is a dict then I think `var_name_tidy` never actually appears. 
Thus the tidying can overwrite `var_name` column and the user need not think about what `var_name_tidy` should be (as they will never see it?)